include/index_helper.cuh

```diff
@@ -121,6 +121,19 @@ namespace quda {
   {
     return linkIndexDn<I, -1>(x, X, mu);
   }
+  
+  /**
+     Compute the checkerboard 1-d index from the 4-d coordinate x[] -2 in the mu direction
+
+     @return 1-d checkerboard index
+     @param x 4-d lattice index
```

Comment:
Missing `[in]` / `[out]` labels from the parameters, and the `@return` should be after the parameter list.